lib/actions/addPrice.js

```diff
@@ -29,17 +29,28 @@ function action(msg, cfg) {
 
     function addPrice(response) {
         var product = response.body;
+        var stagedProduct = product.masterData.staged;
+        var variantId = parseInt(data.variantId, 10);
+
+        var variantObject;
+        if (variantId === 1) {
+            variantObject = stagedProduct.masterVariant;
+        } else {
+            variantObject = _.where(stagedProduct.variants, {id: variantId});
+        }
+
+        var foundPriceInSameScope = variantObject && _.where(variantObject.prices, {currencyCode: data.currency});
```

Comment:
Every customer would appreciate a simple log message what you are currently doing. Something like that:

* Adding new {{data.currency}} price for variant {{variantId}}
* Updating {{data.currency}} price for variant {{variantId}}